apps/gitness/tailwind.config.js

```diff
@@ -1,12 +1,5 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
-  presets: [
-    // require('@harnessio/views/tailwind.config')
-    require('@harnessio/ui/tailwind.config')
-  ],
-  content: [
-    'node_modules/@harnessio/views/src/**/*.{ts,tsx}',
-    'node_modules/@harnessio/ui/src/**/*.{ts,tsx}',
-    './src/**/*.{ts,tsx}'
-  ]
+  presets: [require('@harnessio/ui/tailwind.config')],
+  content: ['node_modules/@harnessio/ui/src/**/*.{ts,tsx}', './src/**/*.{ts,tsx}']
```

Comment:
dropping the css being used from views package could break stuff.. this will need proper validation once merged.